install/helm/gloo/templates/1-gloo-deployment.yaml

```
@@ -34,6 +34,7 @@ spec:
         prometheus.io/path: /metrics
         prometheus.io/port: "9091"
         prometheus.io/scrape: "true"
+        solo.io/ossImage: {{ .Values.gloo.deployment.ossImageTag }}
```

```suggestion
        gloo.solo.io/oss-image-tag: {{ .Values.gloo.deployment.ossImageTag }}
```
kubernetes generally seems to [use kebab-case](https://kubernetes.io/docs/reference/labels-annotations-taints/) for annotations/labels (although admittedly we are already inconsistent with our casing in gloo)

also noticed we use the prefix `gloo.solo.io` elsewhere